blockchain/ethereum/demo/demo pool creator/9b8748db-7f0b-4304-b79a-d65a473f255f.json

```diff
@@ -0,0 +1,299 @@
+{
+  "implementations": {
+    "cranq": {
+      "nodes": {
+        "144e4964-bf4e-48f7-8bb1-11551108e472": {
```

Comment:
Failure while loading prototype. Child prototype definitions may be missing, or the prototype might be corrupted. Please try updating your repositories.

[Error] prototype `f2c2b6c4-ddb5-451d-945e-f37457ccf2e6` of output node `100 thousand x 10^18` (64edc4a0-9179-4d53-b761-9621b8a239ec) cannot be found

Reply:
added yesterday